### PR TITLE
Lnk 2241 implement telemetry in botw java service

### PR DESCRIPTION
**Pull request by arianamihailescu**

### CODE REVIEW

File: Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/MeasureEvalApplication.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.lantanagroup.link.measureeval;
 
+import io.opentelemetry.instrumentation.spring.autoconfigure.EnableOpenTelemetry;
```
Comment (seanmcilvenna) [inline Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/MeasureEvalApplication.java]: Is this an unused import?

File: Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/controllers/MeasureDefinitionController.java [hunk 2]
```diff
@@ -8,55 +8,71 @@
...
 
 import java.util.List;
 
+
 @RestController
 @RequestMapping("/api/measure-definition")
 @PreAuthorize("hasRole('LinkUser')")
 public class MeasureDefinitionController {
+
+    private final Logger LOG = LoggerFactory.getLogger(MeasureDefinitionController.class);
```
Comment (seanmcilvenna) [inline Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/controllers/MeasureDefinitionController.java]: Rename to _logger

File: Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/controllers/MeasureDefinitionController.java [hunk 3]
```diff
@@ -8,55 +8,71 @@
...
+        //tracer =  ExtendedTracer.create(openTelemetry.getTracer(INSTRUMENTATION_NAME));
     }
 
     @GetMapping
     @JsonView(Views.Summary.class)
     @Operation(summary = "Get all measure definitions", tags = {"Measure Definitions"})
-    public List<MeasureDefinition> getAll(@AuthenticationPrincipal UserDetails user) {
+    public List<MeasureDefinition> getAll(@AuthenticationPrincipal PrincipalUser user) {
+
+        LOG.info("Loki really works");
```
Comment (seanmcilvenna) [inline Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/controllers/MeasureDefinitionController.java]: Update to a more meaningful message

**Review by seanmcilvenna — CHANGES_REQUESTED** [review]

**Review by amphillipsLGC — APPROVED** [review]
Looks good!

**Pull request merged**